samples/calendar/index.js

```diff
@@ -0,0 +1,143 @@
+
```

Comment:
We need to have a license header in all of these.  You can use this as an example:
https://github.com/googleapis/google-api-nodejs-client/blob/master/samples/drive/list.js